charts/minecraft-bedrock/values.yaml

```diff
@@ -32,7 +32,8 @@ minecraftServer:
   version: "LATEST"
   # One of: peaceful, easy, normal, and hard
   difficulty: easy
-  # A comma-separated list of player names to whitelist.
+  # A comma-separated list of player names to whitelist with no whitespace.
```

Comment:
Isn't the whitelist file also XUIDs like the permissions file?

Reply:
You can use xuid's or names in the whitelist and the server will fill in the missing data. I'd like to require xuid's like the permissions, but whitelist on java is name-based, so I wanted to keep it the same. I wonder if we should change it in some way to allow both.

Reply:
Since names work, then that seems sufficient. Really it's a shame the permissions file doesn't work the same way with filling in the xuid. One of those indicators the bedrock server just isn't fully baked yet.